corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -1059,8 +1074,23 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
                 break;
             case Const.MULTI_SELECT:
                 isMinimal = style === Const.MINIMAL;
+                if (style) {
+                    isLabel = style === Const.LABEL;
+                    hideLabel = style === Const.LIST_NOLABEL;
+                }
+
                 if (isMinimal) {
                     entry = new MultiDropdownEntry(question, {});
+                } else if (isLabel) {
+                    // ChoiceLabelEntry is usually only for radio-type inputs, but utilizing here because
```

Comment:
`ChoiceLabelEntry` is also used as the actual radio input for select questions so that's where I got a bit confused. I'm going to make this change and update the comment description for `ChoiceLabelEntry` to be totalistic so let me know your thoughts there